### PR TITLE
KOGITO-4720: Update timer examples for Jobs Service with MongoDB

### PR DESCRIPTION
**Pull request by xieshenzh**
Description: MongoDB persistence support for Job Service
Jira: https://issues.redhat.com/browse/KOGITO-4720

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `master` branch!**

- [x] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

</details>

### CODE REVIEW

**Review by cristianonicolai — APPROVED** [review]

**Comment by xieshenzh** [comment]
Jenkins retest this

**Comment by cristianonicolai** [comment]
Jenkins rerun examples tests

File: process-timer-quarkus/README.md [hunk 1]
```diff
@@ -323,12 +323,18 @@ java -Dquarkus.http.port=8085 -jar jobs-service-infinispan/target/jobs-service-i
...
-If you'd like to use PostgresSQL as persistence, start the PostgreSQL server, then start job service with following command
+If you'd like to use PostgresSQL or MongoDB as persistence, start the PostgreSQL or MongoDB server, then start job service with following command
 
+For PostgreSQL:
 ```
 java -Dquarkus.http.port=8085 -Dquarkus.datasource.username={username} -Dquarkus.datasource.password={password} -Dquarkus.datasource.reactive.url=postgresql://{host}:{port}/{db} -Dquarkus.datasource.jdbc.url=jdbc:postgresql://{host}:{port}/{db}  -jar jobs-service-postgresql/target/jobs-service-postgresql-{version}-runner.jar
 ```
 
+For MongoDB:
+```
```
Comment (radtriste) [inline process-timer-quarkus/README.md]:
```suggestion
```sh
```

**Review by radtriste — APPROVED** [review]
one small change

**Review by tiagodolphine — APPROVED** [review]

**Comment by tiagodolphine** [comment]
jenkins retest this

**Comment by cristianonicolai** [comment]
Jenkins rerun examples tests

**Comment by xieshenzh** [comment]
Jenkins retest this

**Pull request merged**